### PR TITLE
Rename to recycling views

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

**Comment by 3lvis** [comment]
artboard2

**Comment by 3lvis** [comment]
# Project structure
screen shot 2018-05-02 at 19 35 00

File: Sources/Recycling/GridViews/Ads/Demo/AdsGridViewDemoViewHelpers.swift [hunk 1]
```diff
@@ -5,8 +5,8 @@
 import FinniversKit
 import Foundation
```
Comment (ulrikus) [inline Sources/Recycling/GridViews/Ads/Demo/AdsGridViewDemoViewHelpers.swift]: Don't need this when FinniversKit is imported

File: Sources/Recycling/GridViews/Markets/Demo/MarketsGridViewDemoViewHelpers.swift [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import FinniversKit
 import Foundation
```
Comment (ulrikus) [inline Sources/Recycling/GridViews/Markets/Demo/MarketsGridViewDemoViewHelpers.swift]: Same here. Don't need Foundation when we have FinniversKit imported.

File: Sources/Fullscreen/Frontpage/Demo/FrontpageViewDemoView.swift [hunk 3]
```diff
@@ -6,25 +6,25 @@ import FinniversKit
 import UIKit
```
Comment (ulrikus) [inline Sources/Fullscreen/Frontpage/Demo/FrontpageViewDemoView.swift]: You don't need UIKit when we have FinniversKit imported.

**Comment by 3lvis** [comment]
@ulrikus Removed unused imports :)

**Comment by axldyb** [comment]
Such better, much wow.
unknown

**Comment by 3lvis** [comment]
That’s correct :) I can take care of it.

**Pull request merged**